### PR TITLE
include ethereum transaction receipt in ethTx adapter run result

### PR DESCRIPTION
**Pull request by se3000**
Also moves the mockgen mocks into the internal directory, so that we can keep them together in one place.

### CODE REVIEW

File: adapters/eth_tx_test.go [hunk 1]
```diff
@@ -321,15 +320,64 @@ func TestEthTxAdapter_Perform_FromPendingConfirmations_ConfirmCompletes(t *testi
...
-	tx, err := store.FindTx(tx.ID)
+	tx, err = store.FindTx(tx.ID)
 	assert.NoError(t, err)
 	assert.True(t, tx.Confirmed)
 	attempts, _ := store.TxAttemptsFor(tx.ID)
 	assert.False(t, attempts[0].Confirmed)
 	assert.True(t, attempts[1].Confirmed)
 	assert.False(t, attempts[2].Confirmed)
 
+	fmt.Println(output.Data.String())
```
Comment (dimroc) [inline adapters/eth_tx_test.go]: ✂️ Remove `fmt.Println`

File: store/tx_manager.go [hunk 2]
```diff
@@ -270,37 +275,41 @@ func (txm *EthTxManager) GetLINKBalance(address common.Address) (*assets.Link, e
 	return (*assets.Link)(balance), nil
 }
 
-// MeetsMinConfirmations returns true if the given transaction hash has been
+// BumpGasUntilSafe returns true if the given transaction hash has been
 // confirmed on the blockchain.
-func (txm *EthTxManager) MeetsMinConfirmations(hash common.Hash) (bool, error) {
-	if !txm.Connected() {
-		return false, ErrPendingConnection
```
Comment (dimroc) [inline store/tx_manager.go]: I feel like we still need to check if we're connected before moving foward in `BumpGasUntilSafe`. Can you walk through why we no longer need `ErrPendingConnection`?

Comment (se3000) [reply store/tx_manager.go]: Good catch. I mistakenly thought that was checked under the hood. It wasn't, so I introduced a helper for it instead.

File: store/tx_manager.go [hunk 3]
```diff
@@ -270,37 +271,41 @@ func (txm *EthTxManager) GetLINKBalance(address common.Address) (*assets.Link, e
...
-	}
-	blkNum, err := txm.GetBlockNumber()
-	if err != nil {
-		return false, err
-	}
-	attempts, err := txm.getAttempts(hash)
+func (txm *EthTxManager) BumpGasUntilSafe(hash common.Hash) (*TxReceipt, error) {
+	blkNum, err := txm.getBlockNumber()
```
Comment (dimroc) [inline store/tx_manager.go]: Perhaps we could rename this to `safeGetBlockNumber` or something so it's not so easily mistaken with the original method?

**Pull request merged**